types/react-native-easy-upgrade/index.d.ts

```diff
@@ -0,0 +1,82 @@
+// Type definitions for react-native-easy-upgrade 0.0
+// Project: https://github.com/react-native-org/react-native-easy-upgrade#readme
+// Definitions by: taoqf <https://github.com/taoqf>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
```

Comment:
Are the few instances of `unknown` here really worth restricting the version to 3.0 and higher? Callback return types are very frequently ignored, so `void` or `any` is common. `any` also seems appropriate for the `lookupInfo` dictionary. If you did that, I think you could take the version all the way down to 2.1, so potentially a lot more people could use these typings.